backend/de.metas.rfq/src/main/java/de/metas/rfq/impl/MailRfqResponsePublisherInstance.java

```diff
@@ -148,7 +148,7 @@ public void publish0(final RfQResponsePublisherRequest request, final RfQReportT
 					to, // to
 					null, // cc
 					null, // bcc
-					ArchiveEmailSentStatus.ofEMailSentStatus(emailSentStatus) // status
```

Comment:
what it the `emailSentStatus` indicates a problem and the mail was not send?

Reply:
changed to use proper status.